.github/actions/azureml-test/action.yml

```diff
@@ -74,48 +74,47 @@ runs:
         python-version: "3.8"
     - name: Install azureml-core and azure-cli on a GitHub hosted server
       shell: bash
-      run: pip install azureml-core azure-cli
+      run: pip install --quiet "azureml-core>1,<2" "azure-cli>2,<3"
     - name: Log in to Azure
       uses: azure/login@v1
       with:
         creds: ${{inputs.AZUREML_TEST_CREDENTIALS}}
     - name: Install wheel package
       shell: bash
-      run: pip install wheel
+      run: pip install --quiet wheel
```

Comment:
Thanks for making this silent, these logs were quite annoying!

Reply:
:-)